compute/cker/include/cker/operation/RmsNorm.h

```diff
@@ -33,37 +33,68 @@ inline void RmsNorm(const RmsNormParams &params, const Shape &input_shape, const
                     const Shape &gamma_shape, const float *gamma_data, const Shape &output_shape,
                     float *output_data)
 {
-  const int32_t batches = MatchingDim(input_shape, 0, output_shape, 0);
-  const int32_t heights = MatchingDim(input_shape, 1, output_shape, 1);
-  const int32_t widths = MatchingDim(input_shape, 2, output_shape, 2);
-  const int32_t channels = MatchingDim(input_shape, 3, output_shape, 3);
+  bool single_gamma = gamma_shape.DimensionsCount() == 1 && gamma_shape.Dims(0) == 1;
```

Comment:
@seockho-kim Could you check whether we need to allow scalar number (it may have `DimensionCount() = 0`) ?

Reply:
> @seockho-kim Could you check whether we need to allow scalar number (it may have `DimensionCount() = 0`) ?

Current model has no gamma(scale), so it's set to default gamma ( [1.0] ) after fusing.
So, no need to allow scalar number, now, I think.

Reply:
AFAIK, `gamma` is some value, not `1.0`. Maybe your _current model_ is not final or actual model. Let's check why `gamma` is `1.0`.

Reply:
> AFAIK, `gamma` is some value, not `1.0`. Maybe your _current model_ is not final or actual model. Let's check why `gamma` is `1.0`.

Current model's RMSNorm pattern (need to fuse) has no scale. 
(https://github.com/Samsung/ONE/issues/13964#issuecomment-2337657385)
So, scale(gamma) is set to 1.0 when it is fused to RMSNorm.

Reply:
I talked with @jinevening who provided us model, and concluded `gamma` is not `1.0`. It has actual values, but it seems to be propagated to successors of fully connected weight during our internal quantization and optimization.